SU-SWS/stanford_profile
# D8CORE-1930: Changing the multirow gutters and removing the fixed widths

# READY FOR REVIEW

# Summary
- This is to correct the problems with the widths and gutters in multi-row on the full width screens.

# Needed By (Date)
- Friday, 6/26

# Urgency
- Medium

# Steps to Test

1. Pull in the changes.
2. Create a page in full width and one with no layout.
3. Add multiple items per row and with mixed kinds of paragraphs.
4. It should be tried with different elements mixed up together.

# Affected Projects or Products
- Core product

# Associated Issues and/or People
- D8CORE-1930
- Removed all the fixed percentage widths. Used flex-grow: 1 instead. Added the max with of 880px for the large breakpoint to account for the margins.
- I tested it on Safari, FF, Chrome on the Mac. Also in IE 11 and Edge on browser stack. Also tried the different breakpoints.
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
